lib/models/node.js

```diff
@@ -64,6 +69,31 @@ function NodeModelFactory (Model, waterline, _) {
                     return !_.isEmpty(catalog);
                 });
             }
+        },
+        addTags: function(id, tags) {
+            return Promise.fromNode(this.native.bind(this))
```

Comment:
Promise.fromNode promisifies this: http://sailsjs.org/documentation/reference/waterline-orm/models/native.  Credit to @benbp for the work: https://github.com/RackHD/on-core/pull/75/files#diff-89e9e9d58a3c4aa21cf6a96f2bde6e15R110